rbs/SigsRewriter.cc

```diff
@@ -13,6 +13,19 @@ namespace sorbet::rbs {
 
 namespace {
 
+const string_view RBS_PREFIX = "#:";
+const string_view ANNOTATION_PREFIX = "# @";
+
+pair<string_view, uint32_t> stripAsciiWhitespaceWithCount(string_view str) {
+    uint32_t prefixWhitespaceCount = 0;
+    size_t start = 0;
```

Comment:
nitpick since we increment it:

```suggestion
    size_t index = 0;
```